proposals/0226-package-manager-target-based-dep-resolution.md

```diff
@@ -118,15 +183,15 @@ let package = Package(
         .executable(name: "irc-sample", targets: ["irc-sample"]),
     ],
     dependencies: [
-       .package(name: "NIO", url: "https://github.com/apple/swift-nio.git", from: "1.0.0"),
+       .package("https://github.com/apple/swift-nio.git", from: "1.0.0"),
```

Comment:
Do you need the `url` label here? If nothing else, the package declarations should be consistent

Reply:
good catch, this is a typo